src/rev_ai/apiclient.py

```diff
@@ -120,17 +133,21 @@ def submit_job_local_file(
         payload = {}
         if skip_diarization:
             payload['skip_diarization'] = skip_diarization
+        if skip_punctuation:
+            payload['skip_punctuation'] = skip_punctuation
         if metadata:
             payload['metadata'] = metadata
         if callback_url:
             payload['callback_url'] = callback_url
         if custom_vocabularies:
             payload['custom_vocabularies'] = custom_vocabularies
+        if speaker_channel_count:
+            payload['speaker_channel_count'] = speaker_channel_count
 
         with open(filename, 'rb') as f:
             files = {
                 'media': (filename, f),
-                'options': (None, json.dumps(payload))
+                'options': (None, json.dumps(payload, sort_keys=True))
```

Comment:
Why sort_keys?

Reply:
I was having trouble getting the tests to pass because our testing sorts in some cases and doesn't in others. This was the best way I could figure out to make it consistent without much change